packages/validators/src/utils/string-utils.ts

```diff
@@ -0,0 +1,408 @@
+import { Validation, mapReplace } from '@ngx-validate/core';
```

Comment:
I think we should add an tsconfig alias similar to the following and import anything from core to validators referring to this alias:

```
"paths": {
  "core": [
    "dist/core",
    "libs/core/src/public_api.ts"
  ]
}
```